app/hotels/src/filter/hotelFacilities/HotelFacilitiesPopup.js

```diff
@@ -98,7 +98,14 @@ export default class HotelFacilitiesPopup extends React.Component<
     facilities: this.props.facilities,
   };
 
-  static getDerivedStateFromProps = ({ facilities }: Props) => ({ facilities });
+  static getDerivedStateFromProps = ({ facilities, isVisible }: Props) => {
+    if (!isVisible) {
```

Comment:
I find it a bit counter-intuitive that `!isVisible` returns non null.

Reply:
Should I rather flip it to `if(isVisble) return null`?

Reply:
Maybe rename the variable to make it clearer why `isVisible` corresponds to `null`. Something like 
```
({facilities, isVisible: isFilterSelectionVisible}:Props) => {
const isPopUpVisible = !isFilterSelectionVisible;
if (isPopUpVisible) {
return ...}
return null;
```

Reply:
I don't agree. The component is named `...Popup` and isVisible should be quite clear what is going on. 

I would also expect that something from outside should not change my popup while I am looking at it. 

I will make the returns a little neater though.